infrastructure/logic-app/rota/parameters-dev.json

```diff
@@ -33,6 +33,21 @@
         },
         "hmiSchedulesUrl":{
             "value": "https://hmi-apim.dev.platform.hmcts.net/hmi/schedules"
+        },
+        "serviceNowHostUrl": {
+            "value": "https://mojcppdev.service-now.com/api/now/table/incident?sysparm_fields=number"
+        },
+        "serviceNowAssignemntGroupId": {
+            "value": "56b756774fbd368057db0b318110c7bd"
+        },
+        "serviceNowUserRoleTypeId": {
+            "value": "c319bc4bdb41834074abffa9bf96199c"
+        },
+        "serviceNowServiceOfferingId": {
+            "value": "138e0c541bc5bc507bdaddf0b24bcb2a"
+        },
+        "serviceNowCallerId": {
+            "value": "1475a0491b743414f0dc85e4464bcb7a"
```

Comment:
Just checking ok with these id's being present?

Reply:
yes - these are env specific for ticket creation